WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java

```diff
@@ -222,4 +235,57 @@ public void onErrorResponse(VolleyError volleyError) {
         AppLog.d(T.READER, "requesting post");
         WordPress.getRestClientUtilsV1_1().get(path, null, null, listener, errorListener);
     }
+
+    private static String getTrackingPixelForPost(@NonNull ReaderPost post) {
+        return "https://pixel.wp.com/g.gif?v=wpcom&reader=1"
+                + "&blog=" + post.blogId
+                + "&post=" + post.postId
+                + "&host=" + UrlUtils.urlEncode(UrlUtils.getDomainFromUrl(post.getBlogUrl()))
+                + "&ref="  + UrlUtils.urlEncode(TRACKING_REFERRER)
+                + "&t="    + mRandom.nextInt();
+    }
+
+    public static void bumpPageViewForPost(long blogId, long postId) {
+        ReaderPost post = ReaderPostTable.getPost(blogId, postId, true);
+        if (post == null) {
+            return;
+        }
+
+        // don't bump stats for posts in blogs the current user is an admin of, unless
+        // this is a private post (we count views for private blogs from admins)
```

Comment:
This should be " this is a private post (we count views for private post from admins)". :)